detect_secrets/core/usage.py

```diff
@@ -210,13 +210,13 @@ def _add_custom_limits(self):
         self.parser.add_argument(
             '--base64-limit',
             type=self._argparse_minmax_type,
-            nargs=1,
+            nargs='?',
```

Comment:
What does it mean to have `--base64-limit` without any limit?

Reply:
I'm not sure I follow, do you mean if I do `python -m detect_secrets.main --base64-limit --scan test_secrets/`? It uses the default.